doc/lua_api.md

```diff
@@ -5869,6 +5869,16 @@ Environment access
       returns `{name="ignore", param1=0, param2=0}` for unloaded areas.
 * `minetest.get_node_or_nil(pos)`
     * Same as `get_node` but returns `nil` for unloaded areas.
+* `minetest.bulk_get_node({pos1, pos2, pos3, ...})`
+    * Get nodes from all positions set in the first argument.
+    * Returns list of nodes.
+    * e.g. `minetest.bulk_get_node({{x=0, y=1, z=1}, {x=1, y=2, z=2}})`
+    * For node specification or position syntax see `minetest.get_node` call
+    * Faster than `get_node` due to single call, but still considerably slower
+      than Lua Voxel Manipulators (LVM) for large numbers of nodes.
+      It also allows getting nodes from spread out positions which would cause
+      LVMs to waste memory.
+      For getting a cube, this is 1.3x faster than `set_node`.
```

Comment:
it doesn't matter too much but: a cube is what? 3x3x3?

Reply:
Updated aslo in bulk_set_node.